src/components/ReferenceDirectoryWithFilter/index.tsx

```diff
@@ -72,18 +72,26 @@ export const ReferenceDirectoryWithFilter = ({
 
     return categoryData.reduce((acc: FilteredCategoryData[], category) => {
       const filteredSubcats = category.subcats.reduce(
-        (subAcc, subcat) => {
+        (subAcc :typeof category.subcats, subcat) => {
           const filteredEntries = subcat.entries.filter((entry) =>
             entry.data.title
               .toLowerCase()
               .includes(searchKeyword.toLowerCase()),
           );
+          if (subcat.entry &&
```

Comment:
Could we format this like the brackets in [this section](https://github.com/processing/p5.js-website/blob/678a03d2092c3e99e571b7713f8007f7908fe37e/src/scripts/parsers/reference.ts#L136-L146), where the parts in the brackets are indented in once? e.g.:
```js
if (
  condition1 &&
  condition2
) {
  doSomething(); // code here, indented in once too
}
```
Currently, I the spacing in the if condition is a tad inconsistent, and the body of the if statement is indented with just one space instead of two.